packages/js/src/ui/components/elements/OptionsDropdown/OptionsDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+import { JSX, For } from 'solid-js';
+import { cn, useStyle } from '../../../helpers';
+import { DotsMenu } from '../../../icons';
+import { Button, Dropdown } from '../../primitives';
+
+export type Option = { label: string; leftIcon?: JSX.Element; rightIcon?: JSX.Element };
+
+const dropdownTriggerButtonVariants = () =>
```

Comment:
Shouldn't this be in `DropdownTrigger.tsx` ?

Reply:
Not sure what you mean, but the idea behind this function is that you can use it when you want to apply the base styles for that and extend.

Reply:
What I'm saying is that I would have to apply it in the dropdown trigger, right? 
And then because we render the `Button` component with `asChild` prop I would have to update the `Button class` prop here as well. Check how the code looks now.